packages/playground/src/dashboard/components/NodeMintingDetails.vue

```diff
@@ -1,65 +1,94 @@
-                      {{ getDateFromTimestamp(receipt.endPeriodTimestamp) }}
-                      <br />
-                      <span class="font-weight-bold">Node TFT Amount :</span> {{ receipt.tft || 0 }} TFT
-                      <span v-if="receipt.fixupReward">+ {{ receipt.fixupReward || 0 }} TFT FixedUp</span>
-                      <br />
-                      <span class="font-weight-bold">Cloud Units :</span>
-                      <ul>
-                        <li inset v-for="(val, key) in receipt.cloud_units" :key="key">
-                          <b style="text-transform: uppercase">{{ key }}:</b> {{ val }}
-                        </li>
-                      </ul>
-                    </v-card-text>
-                  </v-col>
-                  <v-col cols="2">
-                    <v-chip class="ma-4 me-auto" small :color="getChipColor(receipt.type)">{{ receipt.type }} </v-chip>
-                  </v-col>
+                  <v-list class="custom-list" density="compact">
+                    <h3 class="mb-1 font-weight-medium">
+                      Minting Details<v-chip class="ma-2 me-auto" small :color="getChipColor(receipt.type)"
+                        >{{ receipt.type }}
+                      </v-chip>
+                    </h3>
+                    <div class="border">
+                      <v-row class="row-style">
+                        <v-col class="py-1" cols="1" sm="2" style="min-width: fit-content">
+                          <v-list-item> Date of Payout :</v-list-item>
+                        </v-col>
+                        <v-col class="py-1">
+                          <v-list-item>
+                            <div style="display: flex; justify-content: end">
+                              {{ getDateFromTimestamp(receipt.endPeriodTimestamp) }}
+                            </div>
+                          </v-list-item>
+                        </v-col>
+                      </v-row>
+                      <v-row class="row-style">
+                        <v-col class="py-1" cols="1" sm="2" style="min-width: fit-content">
+                          <v-list-item> Node TFT Amount :</v-list-item>
+                        </v-col>
+                        <v-col class="py-1">
+                          <v-list-item>
+                            <div style="display: flex; justify-content: end">
+                              {{ receipt.tft || 0 }} TFT<span v-if="receipt.fixupReward"
+                                >+ {{ receipt.fixupReward || 0 }} TFT FixedUp</span
+                              >
+                            </div>
+                          </v-list-item>
+                        </v-col>
+                      </v-row>
+                      <v-row v-for="(val, key) in receipt.cloud_units" :key="key" class="row-style">
+                        <v-col class="py-1" cols="1" sm="2" style="min-width: fit-content">
+                          <v-list-item> {{ key }} :</v-list-item>
```

Comment:
please capitalize CU, SU and NU and add their units. might be helpful to also add a tooltip explaining what they are

Reply:
done I added unit to SU & NU 
for CU there is no clear until but I think the tooltip explains
https://manual.grid.tf/knowledge_base/cloud/cloudunits.html?highlight=Cloud%20Units#cloud-units